markdown/usage/tutorials/step_by_step_institutional_profile.md

```diff
@@ -0,0 +1,617 @@
+
+### Required files
+
+In your branch, we will need to initialise a couple of new files, and update a couple of others.
+
+- **create** an empty file to the `conf/` directory named `<your_cluster_name>.config`
+- **create** an empty file to the `docs/` directory named `<your_cluster_name>.md`
+- **edit** and add your profile name to the `nfcore_custom.config` file in the top-level directory of the clone
+- **edit** and add your profile name to the list of clusters on the `README.md` file in the top-level directory of the clone  under the 'Documentation'
+- **edit** and add your profile name to the GitHub actions `.yaml` file (under `.github/workflows/main.yml`)
+
+### Writing the main config file
+
+First we will edit the main config file under `conf/<your_cluster_name>.config`.
+
+#### params scope
+
+In Nextflow, the `params` block of configuration files is typically used for setting pipeline-level parameters. In the case of institutional configs we will very likely not specify pipeline parameters here, but rather add some useful nf-core specific parameters that apply to all pipelines. See the nf-core/configs README for more information how to define _pipeline specific_ institutional configs.
+
+The most useful first step for testing a new nf-core institutional profile is to add to the params scope the `config_profile_*` series of params. These are nf-core specific parameters that are displayed in the header summary of each run, describing what the profile is and who maintains it. Therefore, you can use this when testing the profile to check the profile was actually loaded.
+
+In the `conf/<your_cluster_name>.config` file, add to a params scope something like the following:
+
+```nextflow
+params {
+  config_profile_description = '<your_cluser_name> cluster profile provided by nf-core/configs.'
+  config_profile_contact = '<your_name> (<your_github_handle>)'
+  config_profile_url = 'https://<institutional_url>.com'
+}
+```
+
+Next, in the same scope, we can also specify the `max_*` series of params. These are used by nf-core pipelines to limit automatic resubmission of resource-related failed jobs to ensure submitted retries do not exceed the maximum available on your cluster. These values should be the ones you found for the largest node of your cluster (i.e., the largest node a user's job can be submitted to). For example:
+
+```nextflow
+params {
+  config_profile_description = '<your_cluser_name> cluster profile provided by nf-core/configs.'
+  config_profile_contact = '<your_name> (<your_github_handle>)'
+  config_profile_url = 'https://<institutional_url>.com'
+  max_memory = 2.TB
+  max_cpus = 128
+  max_time = 720.h
+}
+```
+
+Finally, if you have a common resource directory for the iGenomes collection of reference genomes, this can can also go in the `params{}` scope.
```

Comment:
```suggestion
Finally, if you have a common resource directory for the iGenomes collection of reference genomes, this can can also go in the `params` scope.
```